pkg/parser/generate_keyword/genkeyword.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"regexp"
+	"strings"
+)
+
+const (
+	ReservedKeywordStart   = "The following tokens belong to ReservedKeyword"
+	UnreservedkeywordStart = "The following tokens belong to UnReservedKeyword"
+	NotKeywordStart        = "The following tokens belong to NotKeywordToken"
+)
```

Comment:
How about "The following tokens belong to TiDBKeyword" are those tokens keywords or not?

Reply:
Comparing these with https://docs.pingcap.com/tidb/stable/keywords seems to indicate that they should be.

Reply:
Fixed in ed488a3a4f6141ba2a57214696930a452179a7db